src/test/java/guitests/guihandles/TaskCardViewHandle.java

```diff
@@ -131,10 +141,19 @@ public boolean isDisplayedCorrectly(int displayedIndex, ImmutableTask task) {
         boolean isCorrectPinDisplay = isPinDisplayCorrect(task);
         boolean isCorrectCompleteDisplay = isCompletedDisplayCorrect(task);
         boolean isCorrectOverdueDisplay = isOverdueDisplayCorrect(task);
-
-        return isCorrectTitle && isCorrectDescription && isCorrectDateText
-                && isCorrectLocation && isCorrectType && isCorrectPinDisplay
-                && isCorrectCompleteDisplay && isCorrectOverdueDisplay;
+        boolean isCorrectCollapsedState = isTaskCardCollapsedStateCorrect();
+
+        //JUnit Assertion Test: To know which test are failing in detail
+        assertTrue(isCorrectTitle);
```

Comment:
Should probably inline all of the method calls - there's really no need to create more variables here